automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeCreationDiskSpec.scala

```diff
@@ -255,31 +250,41 @@ class RuntimeCreationDiskSpec
         implicit0(authToken: AuthToken) <- Ron.authToken()
         runtime <- createRuntimeWithWait(googleProject, runtimeName, createRuntimeRequest)
         clusterCopy = ClusterCopy.fromGetRuntimeResponseCopy(runtime)
-        // validate that saved files and user installed packages persist
-        _ <- IO(withWebDriver { implicit driver =>
-          withNewNotebook(clusterCopy, Python3) { notebookPage =>
-            val createNewFile =
-              """! echo 'this should save' >> /home/jupyter/test.txt""".stripMargin
-            notebookPage.executeCell(createNewFile)
-            notebookPage.executeCell("! pip install simplejson")
-          }
-        })
+
+        // This command is a bit special. It needs to go through 4 layers of interpolation, scala, bash on the GHA node, bash on the VM leo creates, and finally bash within the docker container
+        // Do not mess with the syntax, particularly the quotes
```

Comment:
THIS is the command (sorry, I thought you abstracted it out above)
see other comment re: the special `\\\` syntax and making it clearer about the escape syntax ...

because it is 2 levels deep, maybe there isn't an _easy_ way of putting that knowledge into code.